scenarios/02_Assassins.cfg

```diff
@@ -8,8 +8,9 @@
 [scenario]
     id=02_Assassins
     name= _ "Assassins"
+    # idea: since the title is "Assassins" maybe make use of the "assassin" micro_ai?
```

Comment:
While the names are similar, that AI is for attacking a leader, or another single target. This scenario is about the AI trying to poison lots of your units, so it's a different goal.